plugin/pulpcore/plugin/download/asyncio/factory.py

```diff
@@ -0,0 +1,116 @@
+import aiohttp
+import atexit
+from gettext import gettext as _
+import ssl
+from urllib.parse import urlparse
+
+from .http import HttpDownloader
+
+
+class DownloadFactory(object):
```

Comment:
All objects are new-style in python3.  Don't need the `(object)`.

Reply:
Fixed in next push and throughout PR.